ptf/saiacl.py

```diff
@@ -0,0 +1,5862 @@
+                  src_ip, ' | ', dst_ip, ' | @ ptf_intf 2')
+            send_packet(self, sport, pkt_tcp)
+            # ensure the TCP packet is dropped and check for absence
+            # of packet here
+            print('#### NOT Expecting TCP ', dmac, ' | ', ROUTER_MAC, ' | ',
+                  src_ip, ' | ', dst_ip, ' | @ ptf_intf 1')
+            verify_no_other_packets(self, timeout=2)
+
+            print('#### Sending   UDP', ROUTER_MAC, ' | ', smac, ' | ',
+                  src_ip, ' | ', dst_ip, ' | @ ptf_intf 2')
+            send_packet(self, sport, pkt_udp)
+            # ensure the UDP packet is forwarded
+            print('#### Expecting UDP ', dmac, ' | ', ROUTER_MAC, ' | ',
+                  src_ip, ' | ', dst_ip, ' | @ ptf_intf 1')
+            verify_packets(self, exp_pkt_udp, [dport])
+
+            # change action_type of ACL entry from ACL_DROP to ACL_PERMIT
+            aclaction_data = sai_thrift_acl_action_data_t(
+                parameter=sai_thrift_acl_action_parameter_t(
+                    s32=SAI_PACKET_ACTION_FORWARD), enable=True)
+            sai_thrift_set_acl_entry_attribute(
+                client=self.client,
+                acl_entry_oid=acl_entry_id,
+                action_packet_action=aclaction_data)
+
+            print('#### Sending      ', ROUTER_MAC, ' | ', smac, ' | ', src_ip,
+                  ' | ', dst_ip, ' | @ ptf_intf 2')
+            # send the same packet
+            send_packet(self, sport, pkt_tcp)
+            print('#### Expecting ', dmac, ' | ', ROUTER_MAC, ' | ', src_ip,
+                  ' | ', dst_ip, ' | @ ptf_intf 1')
+            # check that TCP packet is forwarded
+            verify_packets(self, exp_pkt_tcp, [dport])
+
+        finally:
+            # unbind this ACL table from ports object id
+            if table_stage == SAI_ACL_STAGE_INGRESS:
+                sai_thrift_set_port_attribute(
+                    self.client, self.port11, ingress_acl=0)
+            elif table_stage == SAI_ACL_STAGE_EGRESS:
+                sai_thrift_set_port_attribute(
+                    self.client, self.port11, egress_acl=0)
+            # cleanup ACL
+            sai_thrift_remove_acl_entry(self.client, acl_entry_id)
+            sai_thrift_remove_acl_table(self.client, acl_table_id)
+
+    def tearDown(self):
+        sai_thrift_remove_route_entry(self.client, self.route_entry1)
+        sai_thrift_remove_next_hop(self.client, self.nhop1)
+        sai_thrift_remove_neighbor_entry(self.client, self.neighbor_entry1)
+        sai_thrift_remove_route_entry(self.client, self.route_entry2)
+        sai_thrift_remove_next_hop(self.client, self.nhop2)
+        sai_thrift_remove_neighbor_entry(self.client, self.neighbor_entry2)
+        super(IPv6NextHdrTest, self).tearDown()
+
+
+@group("draft")
+@group('acl')
+@group('acl-ocp')
+class IPAclFragmentTest(SaiHelper):
+    """Verify ACL with IP fragmentation"""
```

Comment:
Please use unified style convention in all docstrings - like:
'''
\<description\>
'''